pkg/apis/cluster/v1alpha1/cluster_types_test.go

```diff
@@ -75,6 +75,10 @@ func crudAccessToClusterClient(t *testing.T, cs *clientset.Clientset) {
 	}
 
 	// Test deleting the item for delete requests.
+	actual.Finalizers = nil
```

Comment:
It looks like the comment on line 77 needs to stay with the "client.Delete(...)" call on line 82. Please move these new 4 lines of code above that comment.

Reply:
This whole block is testing that we can delete the Cluster object. See comment [below](https://github.com/kubernetes-sigs/cluster-api/pull/190/files#r191566698)